web/template/fixtures.gohtml

```diff
@@ -36,6 +36,8 @@
     </div>
 
     {{ template "input" (input . "donorEmail" "Donor Email" .DonorEmail "classes" "govuk-input--width-20") }}
+    {{ template "input" (input . "donorFirstNames" "Donor First Names" "Sam" "classes" "govuk-input--width-20") }}
+    {{ template "input" (input . "donorLastName" "Donor Last Name" "Smith" "classes" "govuk-input--width-20") }}
```

Comment:
To make e2e tests easier without adding to the huge pile of Sam Smith cases in Sirius integration.